saleor/account/tests/test_account.py

```diff
@@ -312,3 +314,39 @@ def test_customers_show_staff_with_order(admin_user, channel_USD):
         channel=channel_USD,
     )
     assert User.objects.customers().count() == 1
+
+
+@pytest.mark.parametrize(
+    ("country_area_input", "country_area_output", "is_valid"),
+    [
+        ("Dublin", "Co. Dublin", True),
```

Comment:
How will it work when I provide `dublin`? And how it was behaving previously when I was providing `co. dublin`?  (for both action, the result should be the same)

Reply:
Good remark, default validator does strip() and lower():
```
def _match_choices(value, choices):
    if value:
        value = value.strip().lower()
```
Let me quickly handle it.